Makefile

```diff
@@ -0,0 +1,34 @@
+
+BASE=$(shell pwd)
+OSNAME=$(shell uname)
+
+CFGOPTS += --with-crypto
+CFGOPTS += --enable-magic
+
+ifeq ($(OSNAME),Darwin)
+CFLAGS  += -I/usr/local/include/node
+CFLAGS  += -I/usr/local/include
+LDFLAGS += -L/usr/local/lib
+endif
+
+YARA?=3.9.0
+
+libyara: yara
+
+yara:
+	-rm -rf $(BASE)/build/yara
+	-rm -rf $(BASE)/deps/yara-$(YARA)
+	test -f $(BASE)/deps/yara-$(YARA).tar.gz || curl -L -k https://github.com/VirusTotal/yara/archive/v$(YARA).tar.gz > $(BASE)/deps/yara-$(YARA).tar.gz
```

Comment:
WIth this PR we'll stop depending on system-wide `libyara` version and just fetch the one defined in `YARA` variable above.